_config_cockroachdb_no_cache.yml

```diff
@@ -0,0 +1,2 @@
+remote_include:
```

Comment:
Same here -- can we rename the file to clarify it's just no Jekyll-remote cache?

Reply:
Done.